### ISSUE TITLE
[1.16.3] Ultimate Car Mod Fluid extractor doesn't visually connect.

### ISSUE THREAD

**Issue by ProsperCraft** [Kotori316/FluidTank#58]
![2020-10-08_08 55 35](https://user-images.githubusercontent.com/4427684/95476201-3d8cdd00-0944-11eb-9455-b99e07d61d86.png)

LargeFluidTank-16.0.5
car-1.16.3-1.0.2

**Comment by Kotori316** [comment]
In my environment, pipes are connected to the tank.
![2020-10-09_17 53 40](https://user-images.githubusercontent.com/28705167/95563558-80d87000-0a58-11eb-8c3b-f244f4155106.png)
Are you in multiplayer server world?

## Mods:
- forge-1.16.3-34.1.14-client.jar Minecraft {minecraft@1.16.3 DONE}
- LargeFluidTank-16.0.5.jar FluidTank {fluidtank@16.0.5 DONE}
- car-1.16.3-1.0.2.jar Ultimate Car Mod {car@1.16.3-1.0.2 DONE}
- forge-1.16.3-34.1.14-universal.jar Forge {forge@34.1.14 DONE}
- AdditionalEnchantedMiner-1.16.2-16.1.0.jar QuarryPlus {quarryplus@16.1.0 DONE}
- appliedenergistics2-8.1.0-alpha.3.jar Applied Energistics 2 {appliedenergistics2@8.1.0-alpha.3 DONE}
- jei-1.16.3-7.5.0.43.jar Just Enough Items {jei@7.5.0.43 DONE}

**Comment by ProsperCraft** [comment]
Yes, all of our testing is on multiplayer, you can log in using the TownCraft TWO pack on curseforge, the server is in the multiplayer screen already.  The connected textures mod is present which may be related, I have not tested that.

**Comment by Kotori316** [comment]
I want to ask you 2 basic questions, how to start the server world and which mod is used to show the tooltip(the green box in your screenshot).

I installed your modpack with Twitch and successfully launched the client, but the problem didn't happen so I should create server instance. But I don't know how to start server with modpack. Just copying mods and configs to a new server directory?

2nd question is from my curiosity. I added HWYLA and TOP library to show tank info in their tooltip but neither of them is installed to your pack.

**Comment by ProsperCraft** [comment]
The mod showing what you are looking at is called Look At That, we often try to spotlight mods that don't get any attention because of the well-known mods are always used.  

If you would like to join our discord, we have a section for mod authors only to chat and coordinate with each other and with me as the pack author - https://discord.gg/0YcHRlg0xIojCvGu
PM me when you get in and I can tag you with the VIP tag.

Tom The Geek the mod author of Look At That is in the #mod-author-chat channel.  He is looking into adding visual readouts of volume on the tanks.

So making the pack requires disabling clientside mods.  Unfortunately, forge autodisabling clientside only mods quit in 1.12
We have a server up that updates each pack version, but if you wanted to test on your side, here are the mods to disable.

![image](https://user-images.githubusercontent.com/4427684/96287271-ea97d300-0f9e-11eb-8372-6bb620ef5d52.png)

**Comment by Kotori316** [comment]
I launched server world with TownCraft TWO TEST-0.14 but I got the same result.
![2020-10-21_20 07 24](https://user-images.githubusercontent.com/28705167/96713555-c0e60e00-13db-11eb-8e91-9ce88550e263.png)
![2020-10-21_20 21 54](https://user-images.githubusercontent.com/28705167/96713564-c3e0fe80-13db-11eb-8e21-8aa2c216e410.png)
The screenshots were taken in the multiplayer server world.

Umm...
I think the pipe and extractor use blockstate to change the texture, connected or not. In my environment, as the screenshot shows, pipe has `true` in property(north and south). Does it exist in your world?
